arrow-site/docs/docs/quickstart/setup/README.md

```diff
@@ -57,85 +50,22 @@ dependencies {
 ```groovy
 apply plugin: 'kotlin-kapt'
 
-def arrow_version = "0.11.0"
+def arrow_version = "0.13.0"
 dependencies {
     implementation "io.arrow-kt:arrow-optics:$arrow_version"
-    implementation "io.arrow-kt:arrow-syntax:$arrow_version"
     kapt    "io.arrow-kt:arrow-meta:$arrow_version"
 }
 ```
 
 ##### Arrow Core + Arrow Fx
 
 ```groovy
-apply plugin: 'kotlin-kapt'
-
-def arrow_version = "0.11.0"
-dependencies {
-    implementation "io.arrow-kt:arrow-fx:$arrow_version"
-    implementation "io.arrow-kt:arrow-syntax:$arrow_version"
-    kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-}
-```
-
-##### Arrow Core + Arrow Optics + Arrow Fx
```

Comment:
Is there an updated setup for having opics + core and fx?

Reply:
It's just adding their deps, these matrices of combined options don't make sense IMO to maintain if we just show how they are individually added.